buildlib/tools/builds.sh

```diff
@@ -460,6 +479,9 @@ do
 	# cleanup build dir before the task
 	[ -d "${ucx_build_dir}" ] && rm -rf ${ucx_build_dir}/*
 
-	# run the test
-	$test_name || { azure_log_error "Test failed: $test_name"; exit 1; }
+	if ! $test_name; then
```

Comment:
Following our discussion yesterday, did you check it? Both positive & negative path..

Reply:
As you can see here: https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=87769&view=logs&j=8cc70074-80e3-506d-e80d-a542b02430e7&t=da02301d-00be-54d1-aa97-9bf0b310a3f6
Tests in builds.sh are running as expected.

For negative testing I intentionally caused `build_docs` to fail, as expected, this is what happened:
![image](https://github.com/user-attachments/assets/60913af4-93ad-452b-85d5-dd5757fe8b28)